middleware/mqtt_handler.py

```
@@ -352,36 +296,9 @@ def on_message(client: mqtt.Client, userdata: object, msg: mqtt.MQTTMessage) ->
-                else:
-                    logger.debug(
-                        f"Lock auto-release aborted on {target}: state changed mid-check"
-                    )
-                    return
-            else:
-                logger.info(
-                    f"Ignoring scan on {target} (locked). To unlock: eject the spool "
-                    f"in Mainsail, or POST /api/unlock/{target} on the middleware REST API."
-                )
-                return
+            logger.info(f"Overwriting spool on {target} (clearing lock)")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)